start/scaling-up/preparing-to-scale-up.md

```diff
@@ -132,12 +152,35 @@ desired output, and do not conflict with each other when submitted together. Onc
 you are confident that the jobs will complete as desired, then scale up to submitting 
 the entire set of jobs.
 
-## Avoid Exceeding Disk Quotas
+# Monitoring Job Status and Obtaining Run Information
+
+Gathering information about how, what, and where a job ran can be important for both troubleshooting and optimizing a workflow. The following commands are a great way to learn more about your jobs: 
+
+| Command      | Description |
+| ----------- | ----------- |
+| `condor_q`      | Shows the queue information for your jobs. Includes information such as batch name and total jobs. |
+| `condor_q <JobID> -l` | Prints all information related to a job including attributes and run information about a job in the queue. Output includes `JobDurationCategory`, `ServerTime`, `SubmitFile`, etc. |
+| `condor_q <JobID> -af <AttributeName1> <AttributeName2>` | Prints information about an attribute or list of attributes for a single job using the autoformat `-af` flag. The list of possible attributes can be found using `condor_q <JobID> -l`. |
+| `condor_q -constraint '<Attribute> == "<value>"'   | The `-constraint` flag allows users to find all jobs with a certain value for a given parameter. This flag supports searching by more than one parameter and different operators (e.g. `=!=`).  |
+| condor_q -better-analyze <JobID> -pool <PoolName> | Shows a list of the number of slots matching a job's requirements. For more information, see [Troubleshooting Job Errors] (https://support.opensciencegrid.org/support/solutions/articles/5000639785-troubleshooting-job-errors).|

```

Comment:
I like seeing all these options! For a few of the options, I think it could be helpful to include a more specific example of when you would use this option and what information you could find out. For example, in the 'constraint' line, saying that you could use this to find jobs that had started multiple times. 
On the other hand, that would make the table pretty cluttered, so I'm not wedded to the idea. 

Thoughts? We could also leave as is for now and file an issue to revisit later.

Reply:
Since I am also making a more detailed troubleshooting guide where I where I want to provide examples of when to work with many of these commands, I'll just plan on including this in that guide.